packages/core/contracts/gnosis-zodiac/implementation/OptimisticGovernor.sol

```diff
@@ -160,38 +177,47 @@ contract OptimisticGovernor is Module, Lockable {
      */
     function setLiveness(uint64 _liveness) public onlyOwner {
         // Set liveness for disputing proposed transactions.
-        require(_liveness > 0, "liveness can't be 0");
-        require(_liveness < 5200 weeks, "liveness must be less than 5200 weeks");
+        require(_liveness > 0, "Liveness can't be 0");
+        require(_liveness < 5200 weeks, "Liveness must be less than 5200 weeks");
         liveness = _liveness;
         emit SetLiveness(_liveness);
     }
 
     /**
      * @notice Sets the identifier for future proposals.
-     * @dev Changing this after a proposal is made but before it is executed will make it unexecutable.
      * @param _identifier identifier to set.
      */
     function setIdentifier(bytes32 _identifier) public onlyOwner {
         // Set identifier which is used along with the rules to determine if transactions are valid.
-        require(_getIdentifierWhitelist().isIdentifierSupported(_identifier), "identifier not supported");
+        require(_getIdentifierWhitelist().isIdentifierSupported(_identifier), "Identifier not supported");
         identifier = _identifier;
         emit SetIdentifier(_identifier);
     }
 
     /**
-     * @notice This pulls in the most up-to-date Optimistic Oracle.
-     * @dev If a new OptimisticOracle is added and this is run between a proposal's introduction and execution, the
+     * @notice Sets the Escalation Manager for future proposals.
+     * @param _escalationManager address of the Escalation Manager, can be zero to disable this functionality.
```

Comment:
nit: add a comment here about who can call this and the permissioning related to this. also might be useful to add a small detail on what the implications of the EM are in the context of the OG.